Extension/i18n/chs/package.i18n.json

```diff
@@ -47,12 +48,20 @@
 	"c_cpp.configuration.codeAnalysis.excludeBoolean.markdownDescription": "要与文件路径匹配的 glob 模式。设置为 `true` 或 `false` 以启用或禁用模式。",
 	"c_cpp.configuration.codeAnalysis.excludeWhen.markdownDescription": "对匹配文件同辈进行其他检查。将 `$(basename)` 用作匹配文件名变量。",
 	"c_cpp.configuration.codeAnalysis.runAutomatically.markdownDescription": "如果为 `true`，则代码分析将在文件打开或保存后自动在文件上运行。",
+	"c_cpp.configuration.codeAnalysis.clangTidy.codeAction.showDisable.markdownDescription": "如果为`true`，`禁用'代码操作将在可用时显示(下次运行代码分析时)。使用`禁用'代码操作时，它会将警告代码添加到 `C_Cpp.codeAnalysis.clangTidy.checks.disabled`设置。",
+	"c_cpp.configuration.codeAnalysis.clangTidy.codeAction.showDocumentation.markdownDescription": "如果为`true`，“显示文档”代码操作将在可用时显示(下次运行代码分析时)。",
+	"c_cpp.configuration.codeAnalysis.clangTidy.codeAction.showClear.description": "控制哪些“清除”代码分析问题代码操作选项可用。更改设置以显示更多选项可能需要重新运行代码分析。",
+	"c_cpp.configuration.codeAnalysis.clangTidy.codeAction.showClear.None.description": "不显示“清除”代码操作。",
+	"c_cpp.configuration.codeAnalysis.clangTidy.codeAction.showClear.AllOnly.description": "仅显示“全部清除”代码操作(如果只有一种类型，则显示“清除所有<type>”，如果只有一个问题，则显示“清除此项”)。",
+	"c_cpp.configuration.codeAnalysis.clangTidy.codeAction.showClear.AllAndAllType.description": "如果存在多个问题类型，显示“全部清除”代码操作和“清除所有 <type>”代码操作(或如果<type>只有一个问题，显示“清除此项”)",
+	"c_cpp.configuration.codeAnalysis.clangTidy.codeAction.showClear.AllAndAllTypeAndThis.description": "如果有多个问题类型，显示“全部清除”，如果有多个 <type> 问题，显示“清除所有 <type>”以及显示“清除此项”代码操作",
 	"c_cpp.configuration.codeAnalysis.clangTidy.enabled.markdownDescription": "如果为 `true`，则在 `#C_Cpp.codeAnalysis.runAutomatically#` 为 `true` (默认值)时，将启用并自动运行使用 `clang-tidy` 的代码分析。",
 	"c_cpp.configuration.codeAnalysis.clangTidy.path.markdownDescription": "`clang-tidy` 可执行文件的完整路径。如果未指定，并且 `clang-tidy` 在环境路径中可用，则使用该路径。如果在环境路径中找不到，则将使用与扩展捆绑的 `clang-tidy`。",
 	"c_cpp.configuration.codeAnalysis.clangTidy.config.markdownDescription": "指定 YAML/JSON 格式的 `clang-tidy` 配置: `{Checks: '-*,clang-analyzer-*', CheckOptions: [{键: x, 值: y}]}`。当值为空时，`clang-tidy` 将尝试为其父目录中的每个源文件查找名为 `.clang-tidy` 的文件。",
 	"c_cpp.configuration.codeAnalysis.clangTidy.fallbackConfig.markdownDescription": "指定 YAML/JSON 格式的 `clang-tidy` 配置，以在未设置 `#C_Cpp.codeAnalysis.clangTidy.config#`，并且未找到 `.clang-tidy` 文件: `{Checks: '-*,clang-analyzer-*', CheckOptions: [{键: x, 值: y}]}` 时将其用作回退。",
 	"c_cpp.configuration.codeAnalysis.clangTidy.headerFilter.markdownDescription": "与要从中输出诊断的标头名称匹配的 POSIX 扩展正则表达式 (ERE)。始终显示来自每个翻译单元的主文件的诊断。支持 `${workspaceFolder}` 变量(如果不存在 `.clang-tidy` 文件，则该变量将用作默认回退值)。如果此选项不是 `null` (空)，则将替代 `.clang-tidy` 文件中的 `HeaderFilterRegex` 选项(如果有)。",
 	"c_cpp.configuration.codeAnalysis.clangTidy.args.markdownDescription": "要传递给 `clang-tidy` 的其他命令行参数。这些优先于等效的 `C_Cpp.codeAnalysis.clangTidy.*` 设置。",
+	"c_cpp.configuration.codeAnalysis.clangTidy.useBuildPath.markdownDescription": "如果设置了 `true` 和 `compileCommands`，则 `-p=<build-path>` 参数将传递给 `clang-tidy`，而不是在 `--` 之后传递生成参数。如果未设置环境变量以便可以找到系统包含项，则此操作可能不起作用。",
```

Comment:
@Colengms Build path is a rare case where we want text in `` to be translated -- should we add a comment...will they actually follow the comment though?